src/components/connectAdvanced.js

```diff
@@ -129,7 +138,9 @@ export default function connectAdvanced(
           `or explicitly pass "${storeKey}" as a prop to "${displayName}".`
         )
 
-        this.initSelector()
+        this.state = {
+          selector: this.createSelector()
```

Comment:
I haven't looked at what StrictMode warns about yet.  Does it complain about instance variables?  Is that why we're putting the selector into state?

Reply:
No it doesn't.

This is because `gDSFP` is static, so one can't access instances from it

Reply:
Ahhh, I see it now.  Got it.